### ISSUE TITLE
switching to ruby 3.0.0 brings new error

### ISSUE THREAD

**Issue by denkozlovs** [bo-oz/vimeo_me2#37]
[3] pry(#<Movie>)> vimeo_video = vimeo_client.upload_video(self.video_file)
ArgumentError: wrong number of arguments (given 2, expected 0..1)
from /Users/dk/.rvm/gems/ruby-3.0.0@services/bundler/gems/vimeo_me2-3296c6ba08cf/lib/vimeo_me2/base.rb:27:in `request'

while in ruby 2.6.5 it works fine

what could be broken on the code? I didn't find any clue yet...

**Comment by denkozlovs** [comment]
maybe here
```
lib/vimeo_me2/base.rb:18:        request request_uri(url), params
```
params considered as object, not hash?

**Comment by bo-oz** [comment]
Not sure how changing ruby version would be causing this issue... is the video file valid?

**Comment by denkozlovs** [comment]
it is the very same file in both cases

**Comment by jdufresne** [comment]
@bo-oz Provides information as to why this started happening with Ruby 3: https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/

The fix in #39 is a good approach.
